package/MDAnalysis/coordinates/core.py

```diff
@@ -51,7 +51,8 @@ def get_reader_for(filename, permissive=False, format=None):
         filename of the input trajectory or coordinate file
     permissive : bool
```

Comment:
remove the whole docs for permissive, users should not need to know about anymore.